### PR TITLE
[bashible] Show bash debug output for failed sources steps when bootstrap cluster.

### LINKED ISSUE #2158: [dhctl] Add debug feature to installation process
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

Installation to air-gapped environments require additional debug log.

### Proposed Solution

Add --debug option to dhctl bootstrap. This option adds `set -x` to all bashible steps.

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by name212**
Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>

## Description
Add `-x` parameter for bash source step command if step fail 3 times.

## Why do we need it, and what problem does it solve?
Close #2158

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: bashible
type: feature
summary: Show bash debug output for failed sources steps when bootstrap cluster.
impact_level: default
```

### CODE REVIEW

File: candi/bashible/bashible.sh.tpl [hunk 1]
```diff
@@ -251,6 +252,9 @@ function main() {
       echo ===
       echo === Step: $step
       echo ===
+      {{- if eq .runType "ClusterBootstrap" }}
+      if [ "$attempt" -gt 2 ];then sx=x;fi
```
Comment (RomanenkoDenys) [inline candi/bashible/bashible.sh.tpl:256]:
```suggestion
      if [ "$attempt" -gt 2 ]; then 
        sx="x"; 
      fi
```

Comment (RomanenkoDenys) [reply candi/bashible/bashible.sh.tpl:256]: More readable

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some NIT

File: candi/bashible/bashible.sh.tpl (hunk 1 above)
Comment (name212) [reply candi/bashible/bashible.sh.tpl:256]: I tried to economy some bytes for aws, but yes this code will render for bootstrap first master node only. Thank you, I've fixed it

**Review by yalosev — APPROVED** [review]

**Review by RomanenkoDenys — APPROVED** [review]

**Pull request merged**